game/src/contrib/components/HealthComponent.java

```diff
@@ -76,12 +78,14 @@ public HealthComponent(@DSLContextMember(name = "entity") Entity entity) {
     }
 
     /**
-     * Adds damage, which is accounted for by the system
+     * Adds damage, which is accounted for by the system and creates a damage popup for the hero.
      *
      * @param damage Damage that should be inflicted
      */
     public void receiveHit(Damage damage) {
         damageToGet.add(damage);
+        if (this.getEntity().equals(Game.getHero().orElseThrow()))
```

Comment:
` if (this.getEntity().equals(Game.getHero().orElseThrow()))`

Dann fliegt gleicht alles auseinander, wenn kein Hero da ist. 
`Game.getHero().ifPresent ...` ist hier "richtiger"